### PR TITLE
Improve fr translation

### PR DESCRIPTION
**Pull request by blacksponge**
## Purpose

Improve fr translation

## Context

\-

## Changes

Changes some translation strings.

## How to test this PR

\-

### CODE REVIEW

**Review by pnax — DISMISSED** [review]

File: src/assets/i18n/fr.json [hunk 1]
```diff
@@ -34,14 +34,14 @@
     "Nameservers": "Serveurs DNS",
     "No data for this test": "Il n'y a pas de résultat pour ce test",
     "No result for this query": "Il n'y a pas de résultat pour ce test",
-    "Notice! More info on undelegated test": "Plus d'information sur les test ",
+    "Notice! More info on undelegated test": "Plus d'information sur les tests non délégués",
```
Comment (tgreenx) [inline src/assets/i18n/fr.json:37]: information -> informations

File: src/assets/i18n/fr.json [hunk 2]
```diff
@@ -34,14 +34,14 @@
...
     "Options": "Options",
     "Page Not Found": "Page introuvable",
     "Parent data fetched with success": "Les données parent ont été récupérées avec succès.",
```
Comment (tgreenx) [inline src/assets/i18n/fr.json:40]: parent -> parentes OU du parent

File: src/assets/i18n/fr.json [hunk 3]
```diff
@@ -34,14 +34,14 @@
...
     "Please set the api endpoint": "Merci de contacter l'administrateur, un point d'acces est nécessaire",
```
Comment (tgreenx) [inline src/assets/i18n/fr.json:41]: acces -> accès

File: src/assets/i18n/fr.json [hunk 4]
```diff
@@ -34,14 +34,14 @@
     "Nameservers": "Serveurs DNS",
     "No data for this test": "Il n'y a pas de résultat pour ce test",
     "No result for this query": "Il n'y a pas de résultat pour ce test",
```
Comment (tgreenx) [inline src/assets/i18n/fr.json:36]: Both the same text for those translations, is that intended?

**Review by tgreenx — CHANGES_REQUESTED** [review]

File: src/assets/i18n/fr.json (hunk 2 above)
Comment (matsduf) [reply src/assets/i18n/fr.json:40]: Is "parentes" plural of "parent"? A domain has only one parent domain.

File: src/assets/i18n/fr.json (hunk 4 above)
Comment (blacksponge) [reply src/assets/i18n/fr.json:36]: :shrug:

File: src/assets/i18n/fr.json (hunk 1 above)
Comment (blacksponge) [reply src/assets/i18n/fr.json]: fixed

File: src/assets/i18n/fr.json (hunk 2 above)
Comment (blacksponge) [reply src/assets/i18n/fr.json]: Fixed (I went for @marc-vanderwal proposal)

File: src/assets/i18n/fr.json (hunk 3 above)
Comment (blacksponge) [reply src/assets/i18n/fr.json]: Fixed

**Comment by blacksponge** [comment]
replaced by #373

**Pull request closed without merging**